core/src/main/scala/kafka/consumer/ZookeeperConsumerConnector.scala

```diff
@@ -111,6 +111,15 @@ private[kafka] class ZookeeperConsumerConnector(val config: ConsumerConfig,
   private val zkCommitMeter = newMeter("ZooKeeperCommitsPerSec", "commits", TimeUnit.SECONDS, Map("clientId" -> config.clientId))
   private val rebalanceTimer = new KafkaTimer(newTimer("RebalanceRateAndTime", TimeUnit.MILLISECONDS, TimeUnit.SECONDS, Map("clientId" -> config.clientId)))
 
+  newGauge(
+    "yammer-metrics-count",
```

Comment:
@junrao I think we probably don't need it. I think com.yammer.metrics.Metrics.defaultRegistry() will refer to the same registry instance for any consumer/server started in the program, i.e. it is singleton. If this is true, then this gauage will provide the total number of yammer metrics registered by any consumer and server in the current process, which should not be tagged with client-id. Am I right?